CHANGES.md

```diff
@@ -12,6 +12,8 @@
 - Fix crash when whitespace followed a backslash before newline in a docstring (#4008)
 
 - Fix crash on formatting code like `await (a ** b)` (#3994)
+- Fix a bug that long `case` blocks will not be split into multiple lines, also enabled
```

Comment:
This needs to be changed only in the preview style.

Reply:
I have now added a fix for the `case case:` issue. Since there will be no user-visible format changes, does this still need to be moved to a preview style?

Reply:
Your change changes formatting (in that lines that previously weren't split now are), so it should go into the preview style. (You should add a new entry to the Preview enum and make your change apply only if that preview feature is on.)

Reply:
Ohh gotcha, thanks! I've now moved it under preview style!